cheroot/ssl/__init__.py

```diff
@@ -113,3 +114,8 @@ def get_environ(self):
     def makefile(self, sock, mode='r', bufsize=-1):
         """Return socket file object."""
         raise NotImplementedError  # pragma: no cover
+
+    def _prompt_for_tls_password(self) -> str:
+        """Define interactive prompt for encrypted private key password."""
```

Comment:
(because this does actual prompting)
```suggestion
        """Prompt for encrypted private key password interactively."""
```